docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -52,29 +52,36 @@ image::images/customize-event-renderer-page.png[Shows the Event Renderer page]
 [float]
 [[view-alert-details]]
 === View alert details
-To further inspect an alert, click the *View details* button from the Alerts table. The Alert details flyout appears and provides several options to view alert data.
+To further inspect an alert, click the *View details* button from the Alerts table. The alert details flyout appears with several options to view alert data.
 
 [role="screenshot"]
-image::images/view-alert-details.png[Shows the Event Renderer button, 200]
+image::images/view-alert-details.png[Shows the Event Renderer button, 300]
 
-The Alert details flyout contains the following informational tabs:
+The alert details flyout contains the following informational tabs:
 
 [discrete]
-*Summary*
+*Overview*
 
-The *Summary* tab provides an overview of the alert and shows relevant threat intelligence details. Use this information to analyze what generated the alert, so you can triage its relevance and resolve it as appropriate.
+The *Overview* tab provides an overview of the alert and shows relevant threat intelligence details. Use this information to analyze what generated the alert, so you can triage its relevance and resolve it as appropriate.
 
-The *Summary* tab always begins with a list of general alert details. These details are conveyed as a curated list of fields containing essential and available information about the alert. Within this section, you can find the alert's current status, locate the rule that produced the alert, learn the alert's severity, and more. You can use these details to start forming a sense of what caused the alert and then continue your investigation by checking out available threat intelligence on the alert.
+The *Overview* tab has two main sections: *Reason* and *Document Summary*. In the *Reason* section, you'll find a custom statement describing what caused the alert and available alert details. You can use this information to quickly understand the alert's origin and to determine whether the alert is relevant to your investigation.
 
-{es-sec} collects threat intelligence data for all alerts. Fields that show threat intelligence data are appended with an informational icon and placed after general alert details on the *Summary* tab.
+TIP: You can view more details about the rule that produced the alert by clicking *View Rule detail page*.
 
 [role="screenshot"]
-image::images/threat-intel-fields.png[Shows a field with threat intel]
+image::images/reason-section.png[Shows the Reason section of an alert]
+
+In the *Document Summary* section, you can find general alert details, including any threat intelligence that's available for the alert. General alert details are conveyed as a curated list of fields containing essential and available information about the alert. You can use these details to start exploring what caused the alert and then continue your investigation by checking out available threat intelligence.
```

Comment:
Be careful when using common American terminology like "checking out" - we should aim to be more direct. Also, don't know if it's worth mentioning the `+` and `-` options in the doc summary to filter the Alerts table. 
 
```suggestion
The *Document Summary* section provides general alert details, including any available threat intelligence data.  Use this information to investigate what generated the alert. 
```

Reply:
@joepeeples have you already doc'd, or are you planning to doc, the new `+` and `-` options somewhere within this topic? If you're not/haven't, I can insert something in the Document Summary section.